splunk_add_on_ucc_framework/commands/rest_builder/user_defined_rest_handlers.py

```diff
@@ -270,6 +287,27 @@ def oas_paths(self) -> Dict[str, oas.PathItemObject]:
         else:
             raise ValueError(f"Unsupported handler type: {self.handlerType}")
 
+    @property
+    def endpoint_registration_entry(self) -> Optional[EndpointRegistrationEntry]:
+        if not self.registerHandler:
+            return None
+
+        if not self.registerHandler.get("actions") or not self.registerHandler.get(
+            "file"
+        ):
+            return None
+
+        file: str = self.registerHandler["file"]
+
+        if file.endswith(".py"):
```

Comment:
What if file doesn't end with `.py`, is that validated before in the build process or in `schema.json`?